### ISSUE TITLE
Entry point file Main.elm can't be elmx

### ISSUE THREAD

**Issue by bbugh** [vic/elmx-webpack-preloader#2]
I'm new to webpack and I've set up a simple project for myself that uses elmx. However, having reviewed the code of this library and trying a bunch of different configurations, it seems like the initial entry Elm file is expected to be `.elm`, like `Main.elm` instead of `Main.elmx`. Is that a limitation of the current implementation of this library, or is it an issue with my setup?

**Comment by vic** [comment]
Yes it's currently a limitation, the preloader actually reads `.elm` files, and then looks inside of them for any `elmx` dependencies to compile.
